### PR TITLE
Minor documentation enhancements

### PR DESCRIPTION
**Pull request by JochenHiller**
* use major version in xargs file

Signed-off-by: Jochen Hiller <jochen.hiller@telekom.de>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,27 +27,25 @@ mvn clean install
...
-You can find the created distribution under **/target/smarthome-packaging-sample-[version].zip**
+You can find the created distribution under **/target/smarthome-packaging-sample-<version>.zip**
 
 ### The directory structure of the distribution
 * **addons**: Folder for hotdeployment of bundles
 * **runtime**: Contains the runtime
- * **concierge**: Contains the conciege osgi runtime
-    * **bundles**: Additional concierge bundles
-    * **framework**: The Framework
+  * **concierge**: Contains the conciege osgi runtime
```
Comment (maggu2810) [inline README.md]: osgi => OSGi

File: README.md [hunk 2]
```diff
@@ -27,27 +27,25 @@ mvn clean install
...
     * **system**: Contains commons and 3rd party bundles
-      * **org.eclipse.jetty**: All jetty bundles
-      * **org.eclipse.smarthome**: All SmartHome bundles
- * **etc**: Quartz configuration, Jetty configuration, keystore
+        * **org.eclipse.concierge**: Concierge OSGi Framework and all Concierge bundles
+        * **org.eclipse.smarthome**: All SmartHome bundles
+        * **org.eclipse.jetty**: All jetty bundles
```
Comment (maggu2810) [inline README.md]: jetty => Jetty

File: pom.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
     <groupId>org.eclipse.smarthome</groupId>
     <artifactId>smarthome-packaging-sample</artifactId>
-    <version>0.9.1-SNAPSHOT</version>
+    <version>0.9.2-SNAPSHOT</version>
```
Comment (maggu2810) [inline pom.xml:8]: Why not `0.10.0-SNAPSHOT`?

Comment (JochenHiller) [reply pom.xml:8]: As ESH is yet at `0.9.0` ? And only minor changes (patch) in docs and script

Comment (maggu2810) [reply pom.xml:8]: Ah, sure, as long as we are using the release it makes sense.

```
<esh.version>0.9.0</esh.version>
```

**Pull request merged**